dev-tools/scripts/releaseWizard.yaml

```diff
@@ -1491,13 +1496,13 @@ groups:
         cmd: ant clean
       - !Command
         cmd: python3 -u dev-tools/scripts/addBackcompatIndexes.py --no-cleanup  --temp-dir {{ temp_dir }} {{ release_version }}  && git add lucene/backward-codecs/src/test/org/apache/lucene/index/
-        logfile: add-bakccompat.log
+        logfile: add-backcompat.log
       - !Command
-        cmd: git diff
+        cmd: git diff --staged
         comment: Check the git diff before committing
         tee: true
       - !Command
-        cmd: git add -u .  && git commit -m "Add back-compat indices for {{ release_version }}"  && git push
+        cmd: git commit -m "Add back-compat indices for {{ release_version }}"  && git push
```

Comment:
Do you remember why this change is needed?

Reply:
Because you already do `git add` on line 1498.